practices/team_culture.md

```diff
@@ -0,0 +1,87 @@
+
+**TL;DR**
+
+Team Culture is an essential part of any organization and it emphasizes on achieving a common goal by sharing values, beliefs, attitudes and behaviours among team members.
+
+## What Is Team Culture?
+
+Team Culture is not dependant on or created by an individual member's personality, but rather built on how the team collectively functions.
+
+The definition of team culture is fairly simple as it implies each member of a team should work towards achieving a common goal with collective efforts, discussions and feedback.
+
+## Why You Might Want Team Culture?
+
+An individual person working on the project is far less productive than a team working on the same project. It is very easy to understand that why a team will be more productive than an individual. An individual person has to do all the work during all stages of development.
+
+Team Culture is the change in traditional workplace culture. It is about achieving a goal through collective effort of all the team members rather than leaving all the burden on an individual person. Team work is very essential part of team culture in every aspect.
+
+_"Teamwork is the ability to work together toward a common vision. The ability to direct individual accomplishments toward organizational objectives. It is the fuel that allows common people to attain uncommon results." -Andrew Carnegie_
+
+![Team culture example](/files/team_culture.jpg)
+[Team culture example](https://pixabay.com/photos/work-office-team-company-internet-5382501/)
+
+## Problems Team Culture Helps to Solve
+
+- Performance
+- Efficiency
+- Aggressive Competition
+- Demotivated Individuals
+- Project Deadline
+- [Meaningless work](/problems/meaningless-work)
+- Lack of ideas
+
+## How to Implement Team Culture?
+
+Implementing team culture at first might be not easy but as you communicate with your co-workers and form a team, you get better at it. Organizing team meetings and discussion helps a lot in further developing team culture at your workplace.
+
+### Create Team Leaders
+
+Leader is a very important person in any team who plays a very crucial role in promoting team culture among members. The leader needs to lead the team by establishing trust among members. The leader must also know how to create enthusiasm in the team so that every member is motivated towards achieving the team's goal.
+
+### Meet Frequently
+
+Meeting frequently with all team members will definitely help in building good team culture. These meetings will help in understanding the team dynamics. It will encourage members to trust each other and build healthy relationships.
+
+### Constructive Feedback
+
+Constructive feedback is one of the best ways to help your team to improve. Avoid overdoing it, keep it simple so that a team member is not burdened by your words.
+Be specific with each team member, every team member is different and providing a specific feedback will help in improving team's productivity and thus improve team culture.
+
```

Comment:
Do you have your favorite way of giving feedback? if so, could you mention it? I, for example, like the burger/sandwich method (good - bad -good). I would love to read your thoughts on this one

Reply:
I agree with you @ridlees on your sandwich method. Because I kind of use it in some way or other. I also believe in starting a feedback on a positive note and ending it in same way as It helps in not making other person feel distracted from the positive aspects into just thinking about the negative ones. So , do you want me to include my thoughts on feedback in this article?

Reply:
If you could, it would be perfect.